### PR TITLE
Update intros.md

### PR DESCRIPTION
**Pull request by Kvaibhav01**
Added my information

### CODE REVIEW

**Comment by rahulkumaran** [comment]
@Kvaibhav01 please reposition your details and also, change the s.no

**Comment by Kvaibhav01** [comment]
@rahulkumaran done.

**Comment by rahulkumaran** [comment]
@Kvaibhav01 still some issues.
Could you pull the changes again? Will merge your PR next so that no conflicts arise.

File: hacktoberfest/intros.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 | 1 | Rahul Arulkumaran | <a href="https://github.com/rahulkumaran">rahulkumaran</a> | Hyderabad, India | Python, Full Stack, ML, Data Science |
 | 2 | Juan Luis C. | <a href="https://github.com/luizhdz">Luiz Hdz</a> | Monterrey, Mexico | Python, JS, Ruby, PHP |
 | 3 | André Monteiro | <a href="https://github.com/AndreAlbuquerque9">André Albuquerque</a> | Pernambuco, Brazil | JS, Java, C# |
-| 4 | Tim Ruiterkamp | <a href="https://github.com/timruiterkamp">Tim Ruiterkamp</a> | Amsterdam, Netherlands | Javascript, node |
-| 5 | Sean B | <a href="https://github.com/seanysean">seanysean</a> | New Brunswick, Canada | HTML, CSS |
+| 4 | Sean B | <a href="https://github.com/seanysean">seanysean</a> | New Brunswick, Canada | HTML, CSS |
+| 5 | Tim Ruiterkamp | <a href="https://github.com/timruiterkamp">Tim Ruiterkamp</a> | Amsterdam, Netherlands | Javascript, node |
+| 6 | Sean B | <a href="https://github.com/seanysean">seanysean</a> | New Brunswick, Canada | HTML, CSS |
```
Comment (rahulkumaran) [inline hacktoberfest/intros.md]: 6 and 4 are repetitive. can you remove the duplicate entry

**Comment by Kvaibhav01** [comment]
@rahulkumaran done

**Pull request merged**